### DISCUSSION TITLE
please unlock an option to use 100 percent of the processor and gpu

### DISCUSSION THREAD

**Discussion by userdenuser** [skyline-emu/skyline#1617] [Game Requests]
I discovered that if with an app to select cpu and gpu frequencies with root you can get up to twice the performance in some games, in my snapdragon 855 I got the performance of an 865, it seems that the emulator only takes advantage of the performance of the cores of gpu and cpu without raising frequencies
excuse my english use google translator i speak spanish

**Comment by Allatel** [comment]
Did you mean overclock? Or game turbo thing?

**Comment by yhak22** [comment]
overclocking the cpu and gpu isn't a good idea it will cause overheating the phone and it may break the phone.
i don't recommend this option.

**Reply by userdenuser** [reply]
I did not overclock, just put the frequencies
to the maximum as if the emulator asked for more resources and the result was almost double the performance

in games like guns gore and canoli i go from 40 fps to 60 without getting hot
Checking the "disable game throtthling" option worked up to 120 fps

**Reply by userdenuser** [reply]
I don't want to bother you, it's very big and care what you do and I'm just ignorant
just notice that the emulator does not ask the processor for more power when it does not reach 60 fps and when I manually set the frequencies to the maximum, the same without asking the processor much, it reaches 60 fps
doing the same in games like genshin impac that if it demands 100 percent then my phone does get quite hot
thanks for everything you do definitely skyline is the best :)

**Reply by yhak22** [reply]
disable game throtthling is for benchmark only not recommend for play games cause it may result in some games a broken texture or crash games.
the app is in early stage so it will be better and better so wait for updates ;)
